carRental/src/pages/Request/CarGroupTable.jsx

```diff
@@ -0,0 +1,61 @@
+const CarGroupTable = ({ numberOfCarGroups }) => {
```

Comment:
Add a comment on top of this component, that it's dummy and that's why we didn't choose to refactor it.

Reply:
done